src/emmetHelper.ts

```diff
@@ -493,10 +493,6 @@ export function isAbbreviationValid(syntax: string, abbreviation: string): boole
 		return false;
 	}
 	if (isStyleSheet(syntax)) {
-		// Fix for https://github.com/Microsoft/vscode/issues/1623 in new emmet
```

Comment:
Does this cause 1623 to show up again? Or is that case now being handled by the changes to `isExpandedTextNoise`?

Reply:
This check has been moved to https://github.com/microsoft/vscode/pull/118666/files